src/IdelOption.py

```diff
@@ -16,17 +16,17 @@ def desciption(player):
         action = str(input(">Do you want to look, walk around, open you inventory or continue? \n"))
         action = re.sub("\n", "", action)
         action = re.sub("\s", "", action)
-        if action.upper() == "LOOK":
+        if action.upper() == "LOOK" or action.upper() == "look":
```

Comment:
for action.upper() the result will always be in CAPS
if ur input is look the result will be LOOK
if ur input is Look the result will be LOOK
therefor the or statement is unnessessary

Reply:
same for all others

Reply:
alrighty